elastic_appsearch.theme.inc

```diff
@@ -95,29 +106,28 @@ function template_preprocess_engine(array &$variables) {
   $rows[] = Common::deepCopy($row);
 
   $label = "Documents Processed";
-  $info = ($indexed_count)? $indexed_count : $engine_info['document_count'];
+  $info = ($indexed_count) ? $indexed_count : $engine_info['document_count'];
   $rows[] = Common::deepCopy($row);
 
   $label = "Documents Index";
   $info = $engine_info['document_count'];
   $rows[] = Common::deepCopy($row);
 
   $failed = ($total_count - $engine_info['document_count']);
-  if($failed){
+  if ($failed) {
     $label = "Failure Count";
     $info = $failed;
     $rows[] = Common::deepCopy($row);
   }
 
-  
   $label = t('Server');
-  $info =  Link::fromTextAndUrl($engine->getServer(), $engine->getServerInstance()->toUrl('canonical'));
+  $info = Link::fromTextAndUrl($engine->getServer(), $engine->getServerInstance()->toUrl('canonical'));
   $rows[] = Common::deepCopy($row);
```

Comment:
Any reason why are we calling `Common::deepCopy($row)` multiple times in this function? By default, array assignment is a copy operation.

Reply:
logic changed